client/src/app/app.component.html

```diff
@@ -53,4 +53,5 @@ <h3>{{ message.summary }}</h3>
 <ng-template [ngIf]="isUserLoggedIn()">
   <my-welcome-modal #welcomeModal></my-welcome-modal>
   <my-instance-config-warning-modal #instanceConfigWarningModal></my-instance-config-warning-modal>
+  <my-custom-modal #customModal></my-custom-modal>
```

Comment:
Should be outside this block because a plugin may want to use the modal even if the user is not logged in